sdk/typescript/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+src/
+dist/
```

Comment:
Going back on what I said on the call -- I think it'll be easier to generate the code in ci and not have it committed to the repo